tests/test_experiment.py

```diff
@@ -611,3 +620,111 @@ def test_experiment_stop_does_not_raise_on_unknown_job_id(
     assert stat == InvalidJobStatus.NEVER_STARTED
     after_cancel = exp.get_status(*all_known_ids)
     assert before_cancel == after_cancel
+
+
+@pytest.mark.parametrize(
+    "job_list",
+    (
+        pytest.param(
+            [
+                (
+                    job.Job(
+                        Application(
+                            "test_name",
+                            exe="echo",
+                            exe_args=["spam", "eggs"],
+                        ),
+                        launch_settings.LaunchSettings("local"),
+                    ),
+                    Ensemble("ensemble-name", "echo", replicas=2).build_jobs(
+                        launch_settings.LaunchSettings("local")
+                    ),
+                )
+            ],
+            id="(job1, job2)",
```

Comment:
Nit: `Ensemble.build_jobs` actually returns a tuple of jobs. This ID might be more accurate if changed to
```suggestion
                (
                    job.Job(
                        Application(
                            "test_name",
                            exe="echo",
                            exe_args=["spam", "eggs"],
                        ),
                        launch_settings.LaunchSettings("local"),
                    ),
                    Ensemble("ensemble-name", "echo", replicas=2).build_jobs(
                        launch_settings.LaunchSettings("local")
                    ),
                )
            ],
            id="(job1, (job2, job_3))",
```